tests/gql_behave/tests/memgraph_V1/features/create.feature

```diff
@@ -175,3 +175,15 @@ Feature: Create
             CREATE (a:A) CREATE (a:B)
             """
         Then an error should be raised
+
+    Scenario: CREATE via CALL:
```

Comment:
just curious, what was the output before in this query?

Reply:
oh ok an exception is thrown that a query should return results.

Reply:
Yeah, we were qualifying this whole thing as READ
```
CREATE (a:A {id: 1});
MATCH (a:A {id: 1})
WITH a
UNWIND range(1,1000) as id
CALL {
    WITH id, a
    CREATE (b:B {prop: 1})
    CREATE (b)-[:IN]->(z)
};
```
even though the 
```
    WITH id, a
    CREATE (b:B {prop: 1})
    CREATE (b)-[:IN]->(z)
```
was correctly categories as a WRITE.
We were simply ignoring the subcall. Now we save the subcall's READ/WRITE status and apply to the higher level query.